src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -145,6 +180,7 @@ public async Task<ICredential> GetCredentialAsync(InputArguments input)
             }
             else
             {
+                _context.Trace.WriteLine($"    Found refresh token: {refreshToken} ");
                 // TODO: should we try and compute if the AT has expired and use it?
```

Comment:
Can now drop this TODO comment, and the one below about "the AT is likely expired" since we're now actually checking and using the AT!